fix-point/stack.yaml

```diff
@@ -1 +1,6 @@
 resolver: lts-11.16
+
+extra-deps:
+  - filesystem-trees-0.1.0.6
+  - directory-1.2.7.1
+  - time-1.7.0.1
```

Comment:
I think it's better to remove all `extra-deps` from here. After you remove `filesystem-trees` from `.cabal` file, there won't be any need to have any of these is `stack.yaml`.